auth/README.md

```diff
@@ -197,14 +197,27 @@ startActivityForResult(
     RC_SIGN_IN);
 ```
 
+If anonymous user account conversion to permanent is required:
```

Comment:
I would expand this sentence as there are a lot of advanced concepts at play here.  Something like this:

"The default FirebaseUI sign-in flow shows UI to either create a new account or sign into an existing account.  If you are using anonymous authentication (link) in your application before calling FirebaseUI, you may want to link the anonymous account to the permanent account the user selects in the UI flow."